vendor/github.com/containers/image/docker/archive/dest.go

```diff
@@ -19,15 +19,31 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
```

Comment:
@mrunalp She's open a pull request on that, but I wanted to get this reviewed now.  We will not look to merge until the containers/image gets merged and vendored into CRI-O